### PR TITLE
Create Examples for cyber4all/orb

### PR DESCRIPTION
**Pull request by Cwagne17**
Completes [Story 8650](https://app.shortcut.com/clarkcan/story/8650/implement-examples-for-the-circleci-cyber4all-orb)

Follows RFC for circleci orb.

### CODE REVIEW

**Review by AStoerkel1 — APPROVED** [review]
Looks good, you might want to explain what's going on with the long bash command to get the tag, or even make it a script since it looks like you use it all over the place

File: src/examples/cicd-eb-docker-service.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        - cyber4all/publish-docker:
+            context:
+              - DockerHub
+            image: "$CIRCLE_PROJECT_REPONAME"
+            tag: lastest
+            check-version: false
+            deploy: true
+            filters:
+              branches:
+                only: releases
```
Comment (JosueMolinaMorales) [inline src/examples/cicd-eb-docker-service.yml:64]: Is there a reason for this being repeated?

**Review by JosueMolinaMorales — COMMENTED** [review]
One thing

**Review by AStoerkel1 — APPROVED** [review]
nice job

**Review by EVogel1999 — APPROVED** [review]

**Review by StarChaser44 — APPROVED** [review]

**Pull request merged**